### PR TITLE
change wait() to await.

### PR DESCRIPTION
**Pull request by alexpantyukhin**

### CODE REVIEW

File: src/Proto.Persistence/Persistence.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public void Init(IProvider provider, IContext context)
             if (t != null)
             {
                 EventIndex = t.Item2;
-                Context.ReceiveAsync(t.Item1).Wait();
+                await Context.ReceiveAsync(t.Item1);
```
Comment (rogeralsing) [inline src/Proto.Persistence/Persistence.cs]: The method is a void method so that would not compile.
Are you missing any additional changes in the commit?

Comment (alexpantyukhin) [inline src/Proto.Persistence/Persistence.cs]: Interesting.. there is the same: https://github.com/AsynkronIT/protoactor-dotnet/blob/master/src/Proto.Persistence/Persistence.cs#L48

Comment (rogeralsing) [inline src/Proto.Persistence/Persistence.cs]: No that one is "async Task PersistReceiveAsync" signature, this one is "void Init"

Comment (alexpantyukhin) [inline src/Proto.Persistence/Persistence.cs]: Hm.. but I see 'async Task InitAsync' https://github.com/AsynkronIT/protoactor-dotnet/blob/master/src/Proto.Persistence/Persistence.cs#L21

Comment (rogeralsing) [inline src/Proto.Persistence/Persistence.cs]: hmm, this is what is in the PR:

change

Oh well, Ill merge, if it turns out strange we can always fix it :-)

Comment (alexpantyukhin) [inline src/Proto.Persistence/Persistence.cs]: hm... don't know how is it possible) and it doesn't show that signature was changed.

**Pull request merged**